src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -466,6 +496,11 @@ public static void TestModifyingJsonObjectPrimaryTypes()
             string bigNumber = veryBigConstant.GetString();
             veryBigConstant.SetInt16(123);
             short smallNumber = veryBigConstant.GetInt16();
+
+            // Incrementing age:
```

Comment:
Which is precisely why a good data model stores birthdates, not ages :smile:.

Reply:
I agree, just wanted to see how it would like with our API if someone wants to do this.